scripts/install.sh

```diff
@@ -0,0 +1,22 @@
+#check if verbose flag is used
+if [[ $* == -v ]]
+then
+set -o xtrace
+fi
+
+#create and activate virtual enviroment
+python -m venv backend/env
```

Comment:
`venv` should be listed as a required dependency (like `pip` and `npm`).

Reply:
I have put information about requirability of venv in README